src/stores/SpaceStore.tsx

```diff
@@ -120,8 +120,19 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
      * should not be done when the space switch is done implicitly due to another event like switching room.
      */
     public async setActiveSpace(space: Room | null, contextSwitch = true) {
-        if (space === this.activeSpace || (space && !space?.isSpaceRoom())) return;
-
+        if (space && !space?.isSpaceRoom()) return;
+        if (space && space === this.activeSpace) {
```

Comment:
What if the active space is `null` (the home space)?

Reply:
Oh yes. I'll add a condition for that